packages/contract/lib/utils/index.js

```diff
@@ -80,6 +80,60 @@ const Utils = {
       .filter(log => log != null);
   },
 
+  consoleLog(receiptLogs) {
+    const logEvents = receiptLogs.filter(x => x.event === "_TruffleLog");
```

Comment:
Should we make `_TruffleLog` more specific here? Maybe `_TruffleConsoleLog`? It's possible we may want that namespace later at some point? What do you think?

Reply:
I think that's a sensible suggestion. I'll make an update w/ this.